insert into dtest values('y1234');
ERROR:  42P01: relation "dtest" does not exist
LINE 1: insert into dtest values('y1234');
                    ^
